src/apps/app_muckrock.erl

```diff
@@ -0,0 +1,54 @@
+-module(app_muckrock).
+-export([start/1]).
+-include("ar.hrl").
+
+-define(CHECK_TIME, 60 * 5).
+
+%%% Scrapes and uploads the latest responses received by the Muckrock FOIA
+%%% collection group.
+
+start(WalletFile) ->
+    ssl:start(),
+    Wallet = ar_wallet:load_keyfile(WalletFile),
+    Queue = app_queue:start(Wallet),
+    spawn(fun() -> server(Queue, []) end).
```

Comment:
I'm not so sure about how app_muckrock will be used, but wouldn't it makes sense to populate the `AlreadySeen` argument to `server/2` by using `app_search` to find all already uploaded PDFs?

Reply:
Yes, that would be very nice.